rudder-core/src/main/resources/hooks.d/node-policy-generated/10-cf-promise-check.hook

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Parameters are:
+# - 1: ISO-8601 date/time that identify that policy generation start.
+# - 2: node ID
+# - 3: policies directory
+# - 4: agent type (cfengine-nova or cfengine-community)
+
+case $4 in
+  "cfengine-community")
+    /var/rudder/cfengine-community/bin/cf-promises -f $3/promises.cf
+  ;;
+
+  "cfengine-nova")
+    /var/cfengine/bin/cf-promises -f $3/promises.cf
+  ;;
+
+  *)
+    /bin/true
```

Comment:
i think it should fail if agent type is not defined
if you really want to run /bin/true, then have a specific param, otherwise it's an open door to silent errors